.babelrc

```diff
@@ -1,18 +1,4 @@
 {
- "presets": [
-   "next/babel"
- ],
- "plugins": [
-   [
-     "babel-plugin-styled-components",
-     {
-       "ssr": true,
-       "minify": true,
-       "transpileTemplateLiterals": true,
-       "pure": true,
-       "displayName": true,
-       "preprocess": false
-     }
-   ]
- ]
+  "presets": ["next/babel"],
+  "plugins": [["styled-components", { "ssr": true }]]
```

Comment:
nice - i think i had a lot of copy pasta from other projects without really understanding what i needed

Reply:
Same 😂 I think this is effectively the same, but seemed to help some issue I was seeing